mava/environment_loop.py

```diff
@@ -278,15 +279,15 @@ def step_executor() -> None:
 
             except Exception as e:
                 if self._executor._evaluator:
-                    print(
+                    tf.print(
```

Comment:
Nice idea @DriesSmit, if you can add it to the `mava/systems/trainer.py` too and remove time.sleep(60) since we don't need it anymore!

Reply:
Good point :) We don't need `time.sleep` anymore. I updated the code. Just make sure it works on your side as expected.